packages/scheduler/src/Scheduler.js

```diff
@@ -48,29 +48,35 @@ var currentPriorityLevel = NormalPriority;
 var currentEventStartTime = -1;
 var currentExpirationTime = -1;
 
-// This is set when a callback is being executed, to prevent re-entrancy.
-var isExecutingCallback = false;
+// This is set while performing work, to prevent re-entrancy.
+var isPerformingWork = false;
```

Comment:
I take it most of the changes in here are about removing the React specific immediate flushing semantics and renames. Can you comment on everything that isn't?